client/extensions/woocommerce/woocommerce-services/views/packages/style.scss

```
@@ -208,6 +219,7 @@
 	width: 25%;
 	text-align: right;
 	padding-right: 16px;
+	margin: 0px;
```

(minor) the unit is not required when using `0`
```suggestion
	margin: 0;
```

---

Done!